test/integration/metrics-compliance-static-analysis.py

```
@@ -0,0 +1,547 @@
+                found_rules.append(record_name)
+                
+                # Check openshift: prefix
+                if not record_name.startswith('openshift:'):
+                    print(f"  {Colors.RED}❌ {record_name} missing openshift: prefix{Colors.ENDC}")
+                    all_pass = False
+                else:
+                    print(f"  ✅ {record_name} has openshift: prefix")
+                
+                # Check label removal
+                if 'max without(instance, pod, job, namespace)' in expr:
+                    print(f"     ✅ Removes pod/instance/job/namespace labels")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)